compiling doesn´t work

tryed with VisualStudio, I always run into this problem:
Any idea?


In file included from src\knx\bau57B0.cpp:4:0:
src\knx\bau57B0.h: In constructor 'Bau57B0::Bau57B0(Platform&)':
src\knx\bau57B0.h:26:21: error: 'Bau57B0::_dlLayer' will be initialized after [-Werror=reorder]
     IpDataLinkLayer _dlLayer;
                     ^
src\knx\bau57B0.cpp:15:30: error:   base 'DataLinkLayerCallbacks' [-Werror=reorder]
       DataLinkLayerCallbacks()
                              ^
src\knx\bau57B0.cpp:11:1: error:   when initialized here [-Werror=reorder]
 Bau57B0::Bau57B0(Platform& platform)
 ^
cc1plus.exe: some warnings being treated as errors

![WhatsApp Bild 2024-07-21 um 13 26 41_16fa80e8](https://github.com/user-attachments/assets/08c22eda-014f-47d6-b3db-64906088dff2)

--- a/src/knx/bau57B0.cpp
+++ b/src/knx/bau57B0.cpp
@@ -9,13 +9,11 @@
 using namespace std;
 
 Bau57B0::Bau57B0(Platform& platform)
-    : BauSystemBDevice(platform),
+    : BauSystemBDevice(platform), DataLinkLayerCallbacks(),
       _ipParameters(_deviceObj, platform),
-      _dlLayer(_deviceObj, _ipParameters, _netLayer.getInterface(), _platform, (DataLinkLayerCallbacks*) this),
-      DataLinkLayerCallbacks()
+      _dlLayer(_deviceObj, _ipParameters, _netLayer.getInterface(), _platform, (DataLinkLayerCallbacks*) this)
 #ifdef USE_CEMI_SERVER
-      ,
-      _cemiServer(*this)
+      , _cemiServer(*this)
 #endif
 {
     _netLayer.getInterface().dataLinkLayer(_dlLayer);

